### PR TITLE
Add `--exclude` to pkgmt `format` and pkgmt `lint` 

### LINKED ISSUE #55: Skip read-only directories and files when formatting and linting
When running `pkgmt format` and `pkgmt lint`, sometimes the repository contains some read-only file ([encountered here](https://github.com/ploomber/core/pull/67)). In such a case `pkgmt` should skip that files with a warning and continue checking ahead.

Sample Traceback:
```
***black returned errors.***
PermissionError: [Errno 13] Permission denied: '/home/mohit/ploomber/core/stats/.gitignore'
```

### PR DESCRIPTION
**Pull request by mehtamohit013**
Closes #55

### CODE REVIEW

**Comment by tonykploomber** [comment]
Is this ready for review?

**Comment by mehtamohit013** [comment]
It is still WIP

**Comment by mehtamohit013** [comment]
@edublancas @tonykploomber 
Ready for review

File: src/pkgmt/formatting.py [hunk 1]
```diff
@@ -31,11 +32,14 @@ def find_root():
...
-def format():
+def format(exclude):
     current = find_root()
 
-    print("Running: black .")
-    res = subprocess.run(["black", "."], cwd=current)
+    exclude_str = "|".join(exclude)
+
+    cmd = ["black", ".", "--extend-exclude", exclude_str]
+    print("Running command:", " ".join(map(quote, cmd)))
```
Comment (edublancas) [inline src/pkgmt/formatting.py:41]: please change from print -> click.echo

Comment (edublancas) [reply src/pkgmt/formatting.py:41]: (all other instances as well)

File: tests/test_cli.py [hunk 2]
```diff
@@ -87,31 +87,49 @@ def stuff():
     result_1 = runner.invoke(cli.cli, ["lint", "tmp_folder1"])
```
Comment (edublancas) [inline tests/test_cli.py:87]: use pytest.mark.parametrize to test each case separately

**Review by edublancas — CHANGES_REQUESTED** [review]
tested it and it works well. just minor comments

**Comment by mehtamohit013** [comment]
@edublancas 
Ready for merge

**Review by edublancas — APPROVED** [review]

**Pull request merged**